festipals/model/pal.js

```diff
@@ -1,15 +1,16 @@
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
+var bcrypt = require('bcryptjs');
 
-var palsSchema = new Schema({
-  first_name: { type: 'String', required: true },
-  last_name: { type: 'String', required: true },
-  e_mail: { type: 'String', required: true },
-  password: { type: 'String', required: true },
+var PalsSchema = new Schema({
+  first_name: { type: 'String' },
+  last_name: { type: 'String' },
+  e_mail: { type: 'String' },
+  password: { type: 'String' },
   pals: { type: Array, default: [] },
   acts: { type: Array, default: [] },
   pendingActInvites: { type: Array, default: [] },
   pendingPalRequests: { type: Array, default: [] },
 });
 
-module.exports = mongoose.model('Pal', palsSchema);
+module.exports = mongoose.model('Pal', PalsSchema);
```

Comment:
Oh damn. Well, I tried to make some routes for signup / login. I didnt finish it through.
http://paste2.org/5X7k64HA